test/integration/globals/default-route.js

```diff
@@ -28,6 +28,26 @@ context('patient page', function() {
       .should('contain', 'worklist/owned-by');
   });
 
+  specify('current clinician has reduced patient schedule access', function() {
```

Comment:
Verifies that the page routes to `”/schedule”` by default if the clinician has `”employee”` access and has the `reduced_patient_schedule = true` setting.